src/lib/Components/Bidding/Components/Button.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import { StyleSheet } from "react-native"
+
+import { ButtonProps, InvertedButton } from "../../Buttons"
+
+export class Button extends React.Component<ButtonProps> {
+  render() {
+    const { style, ...props } = this.props
+
+    return <InvertedButton style={[styles.default, style]} {...props} />
+  }
+}
+
+const styles = StyleSheet.create({
+  default: {
+    // fontSize: 14,
+    // fontWeight: 'normal',
+    // fontStyle: 'normal',
+    // letterSpacing: 1,
+    // color: '#ffffff'
+  },
+})
```

Comment:
Leaving this here since we want to inherit some styles and behaviors from `<InvertedButton>` but it's not a styled component.